src/components/credits/CreditsMusic.jsx

```diff
@@ -0,0 +1,84 @@
+import YouTube from 'react-youtube';
+import { Button } from 'react-bootstrap';
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlay, faStop } from '@fortawesome/free-solid-svg-icons';
+import { toast } from 'react-toastify';
+
+export default class CreditsMusic extends React.Component {
```

Comment:
Please use function components instead of class components: https://reactjs.org/docs/components-and-props.html#function-and-class-components

This helps avoid errors with `this`, makes the codebase more consistent (everything is a function component) and makes it possible to use hooks.

Reply:
Still thinking if we should really use YouTube, because it leads to privacy issues and I don't know if react-youtube uses the `youtube-nocookie` domain (which is required by GDPR). Maybe we should just upload the music ourselves...?

Reply:
Uploading the music ourselves would potentially cause copyright issues, which is why I decided to use a hidden YouTube player.

Reply:
Also, it uses the proper YouTube player for your region. If nothing else, we can include a cookie notice on the front page as we use cookies ourselves for authentication.

Reply:
> Uploading the music ourselves would potentially cause copyright issues, which is why I decided to use a hidden YouTube player.

Hm that's true. I can also take a look at it, but not before the weekend though.

> If nothing else, we can include a cookie notice on the front page

I would rather avoid that since it needs opt-in. Cookies for authentication is fine since they are functional cookies which need no opt-in.